### PR TITLE
Modify documentation to reflect the exclusion of http4e client from IntegrationStudio

### PR DESCRIPTION
**Pull request by Tharsanan1**
- Modified the documentation according to this feature removal. 
- Added video clips and images to guide customers to use Postman as an HTTP client.

### CODE REVIEW

File: en/docs/tutorials/integration-tutorials/sending-a-simple-message-to-a-service.md [hunk 1]
```diff
@@ -296,14 +296,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send the request to the API. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/sending-a-simple-message-to-a-service.md:299]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/sending-a-simple-message-to-a-datasource.md [hunk 2]
```diff
@@ -323,14 +323,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send a request to the API resource to make a reservation. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/sending-a-simple-message-to-a-datasource.md:326]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/routing-requests-based-on-message-content.md [hunk 3]
```diff
@@ -434,16 +434,11 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 #### Send the client request
 
-Let's send a request to the API resource to make a reservation. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
+Let's send a request to the API resource to make a reservation. You can use Postman application as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/routing-requests-based-on-message-content.md:439]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/storing-and-forwarding-messages.md [hunk 4]
```diff
@@ -411,14 +411,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send a request to the API resource. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/storing-and-forwarding-messages.md:414]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/transforming-message-content.md [hunk 5]
```diff
@@ -448,14 +448,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send a request to the API resource to make a reservation. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/transforming-message-content.md:451]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/using-inbound-endpoints.md [hunk 6]
```diff
@@ -124,14 +124,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send a message to the **healthcare** REST API (through the inbound endpoint) on port 8285. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/using-inbound-endpoints.md:127]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/using-templates.md [hunk 7]
```diff
@@ -484,14 +484,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send a simple request to invoke the service. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/using-templates.md:487]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

File: en/docs/tutorials/integration-tutorials/using-the-gmail-connector.md [hunk 8]
```diff
@@ -435,14 +435,9 @@ Let's test the use case by sending a simple client request that invokes the serv
 
 Let's send a request to the API resource. You can use the embedded <b>HTTP Client</b> of WSO2 Integration Studio as follows:
 
-1. Open the <b>HTTP Client</b> of WSO2 Integration Studio.
+1. Open the Postman application. If you do not have the application, download from here : [Postman](https://www.postman.com/downloads/)
```
Comment (Mariangela) [inline en/docs/tutorials/integration-tutorials/using-the-gmail-connector.md:438]:
```suggestion
1. Open the Postman application. If you do not have the application, download it from here : [Postman](https://www.postman.com/downloads/)
```

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@Tharsanan1 , pls incorporate the suggestions, resolve the merge conflict and merge the PR. 
Thanks,
Mariangela

**Comment by Tharsanan1** [comment]
Incorporated the suggestions and resolved the merge conflict

**Pull request merged**